frontend/server/libs/dao/Problemsets.dao.php

```diff
@@ -104,4 +104,41 @@ public static function getWithTypeByPK($problemset_id) {
 
         return $problemset;
     }
+
+    /**
+     * Validates that users don't have made submissions with any of their
```

Comment:
```suggestion
     * Checks whether users have made submissions with any of their
```

esto no está validando (porque no hay un valor válido o inválido), sino obteniendo información.

Reply:
Listo